app/config/processing.js

```diff
@@ -0,0 +1,29 @@
+const Joi = require('joi')
+
+// Define config schema
+const schema = Joi.object({
+  pollingInterval: Joi.number().default(10000), // 10 seconds
+  maxProcessingTries: Joi.number().default(3),
+  useV2Events: Joi.boolean().default(true),
```

Comment:
Something to consider, but renaming this to just `useEvents`  The V2 is redundant as we no longer use the V1 event model.